numpy/polynomial/tests/test_polynomial.py

```diff
@@ -102,6 +103,19 @@ def test_polydiv(self):
                 res = poly.polyadd(poly.polymul(quo, ci), rem)
                 assert_equal(res, tgt, err_msg=msg)
 
+    def test_polypow(self):
+        for i in range(5):
+            for j in range(5):
+                msg = "At i=%d, j=%d" % (i, j)
+                c = list(range(i+1))
+                [c] = pu.as_series([c])
```

Comment:
Why do you need `as_series` here? None of the other tests do it.

Reply:
Perhaps just `c = np.arange(1, i + 1)` in place of both these lines?

Reply:
That sounds good to me. Thanks for the recommendation.